### DISCUSSION TITLE
Welcome to the MISS ELS Discussions!

### DISCUSSION THREAD

**Discussion by matsn0w** [matsn0w/IIS-EmergencyLS-ELS-FiveM#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.
  
  Prefer to talk via Discord? You can [join my 'DM' server](https://matsn0w.dev/discord)!

**Comment by nectar1664** [comment]
im having an error that i cant seem to figure out

<img width="953" alt="Capture" src="https://user-images.githubusercontent.com/97266304/148477266-89bd189e-e823-4150-acea-c9b9c5d2aa1b.PNG">

any help? ive already copied the config file

**Reply by matsn0w** [reply]
Hi there, is your config file exactly called `config.lua`? Otherwise it might have something to do with the fact that you've changed the name of the resource. I'll try to reproduce your issue.

Your ELS folder should look like this:

![image](https://user-images.githubusercontent.com/15019582/148523832-487250bd-a9d7-4972-9f75-a0b32c2d7092.png)

Note that this resource is still a work in progress and hasn't been 'officially' released yet. Coming soon tho!

EDIT: I tried renaming the resource like you did and it worked fine.

**Reply by nectar1664** [reply]
Still having the issue, redownloaded the file and re did everything. You dont happen to have a tutorial do you? there was also mention of a base events file, i didnt see anything named as such

**Reply by matsn0w** [reply]
No, I don't have a tutorial yet. 
The baseevents resource is included with FiveM and should be started automatically as it iis marked as a dependency.

Can I see your full server log? Do you have a Windows or a Linux server?

If you are on Windows, are you 100% sure you have 'show file extentions' enabled in Explorer and the file ends with `.lua`?

**Reply by nectar1664** [reply]
Hey sorry its been a minute, life happened.

Im running a windows server, I do have show file extensions enabled and the file endings are correctly labled .lua

im going to attach my server log, im still kinda new to this so it may be something simple im overlooking

<img width="960" alt="Server 1" src="https://user-images.githubusercontent.com/97266304/149700948-94da435d-f701-4500-a2e0-c260a3bca9ad.PNG">
g

<img width="960" alt="server 2" src="https://user-images.githubusercontent.com/97266304/149700967-1cd71474-12e5-44ea-8429-1b19049c45ef.PNG">

<img width="959" alt="server 3" src="https://user-images.githubusercontent.com/97266304/149700973-a6d42107-b691-4325-ad68-c3ed3127e245.PNG">

<img width="960" alt="server 4" src="https://user-images.githubusercontent.com/97266304/149700987-b789df69-afc6-4d15-8fb7-8922416abc75.PNG">

<img width="960" alt="sererv 5" src="https://user-images.githubusercontent.com/97266304/149700992-22746c82-5cc0-46e2-a711-7c310db9473c.PNG">

**Reply by matsn0w** [reply]
Ah, now I see! There seems to be an issue with your Config.lua file (`line 22: '}' expected (...)`)

Can you post your config? I think you missed a bracket somewhere.

**Reply by nectar1664** [reply]
Sure thing

<img width="957" alt="config1" src="https://user-images.githubusercontent.com/97266304/149804120-7c174c44-c532-4ce8-ad8a-58b20319cad7.PNG">
  
<img width="955" alt="config2" src="https://user-images.githubusercontent.com/97266304/149804147-d8574188-10be-4720-9c64-98e9f10e8173.PNG">

**Reply by matsn0w** [reply]
Bingo, you miss a `}` on line 18!

**Reply by nectar1664** [reply]
Something so simple i missed it lol. It solved that issue but nothing is working still. Now im receiving this error

<img width="686" alt="els1" src="https://user-images.githubusercontent.com/97266304/149806388-b4474ccf-c42b-4b32-aad1-a3269edcd091.PNG">
.

**Reply by matsn0w** [reply]
That's a parse error, meaning that one of your VCF's is not correctly formatted. Please make sure your files are valid! Carefully look for missing characters (like the `}`) or use an XML validator to check the files automatically.

**Reply by nectar1664** [reply]
would that prevent the entire script from working properly or just that one file? as of now nothing is working

**Reply by matsn0w** [reply]
The entire script stops working on a parse error.

**Reply by matsn0w** [reply]
@nectar1664 I think that the script should just continue when a VCF is faulty. I'll add functionality for this later. See #44

**Reply by matsn0w** [reply]
@nectar1664 fixed in #46 !
